### PR TITLE
created login page

### PR DESCRIPTION
**Pull request by suryadas-trenser**

### CODE REVIEW

File: WEATHERAPP/Components/Login/LoginPage.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
+body
+{
+   margin: 0;
+}
+.background
+{
+   height: 100vh;
+   background-image:url("../../Assets/L&D-BG\ 4.png");
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/Login/LoginPage.css]: Add background-size: cover and background-position: center to ensure the image covers the entire container.

File: WEATHERAPP/Components/Login/LoginPage.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+   width: 300px;
+   height: 300px;
+}
+.login-form
+{
+   display: flex;
+   flex-direction: column;
+   align-items: center;
+   justify-content: space-evenly;
+   background-color: rgba(132, 95, 95, 1);
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/Login/LoginPage.css]: Add the colors in a separate file, called colors .css and try to use here. Change it in the entire file

File: WEATHERAPP/Components/Login/LoginPage.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+     width: 100px;
+     height: 15px;
+  }
+  
+  .buttons-section {
+     width: 100px;
+  }
+  
+  .navigating-button {
+    height: 15px;
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/Login/LoginPage.css:105]: Consider adding cursor: pointer and transition properties for better user experience.

File: WEATHERAPP/Components/Login/LoginPage.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
+body
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/Login/LoginPage.css]: Add media queries for:
    - Mobile landscape (e.g., max-width: 768px)
    - Tablet portrait (e.g., max-width: 1024px)
    - Desktop (e.g., min-width: 1280px)

File: WEATHERAPP/Components/Login/LoginPage.html [hunk 5]
```diff
@@ -1,2 +1,31 @@
 <!DOCTYPE html>
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/Login/LoginPage.html:1]: Add event-handler functions for the buttons and inputs

File: WEATHERAPP/Components/Login/LoginPage.html [hunk 6]
```diff
@@ -1,2 +1,31 @@
...
+                        <input type="password" class="input-fields" name="password" id="password" placeholder="Password" aria-label="password" required>
+                    </section>
+                    <section class="buttons-section">
+                        <button class="navigating-button" id="sign-in">Sign In</button>
+                        <button class="navigating-button" id="sign-up">Sign UP</button>
+                    </section>
+                </section>
+            </main>
+        </div>
+    </body>
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/Login/LoginPage.html]: Wrap input fields and button in a form element for better semantic structure.

File: WEATHERAPP/Components/Login/LoginPage.css (hunk 1 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/Login/LoginPage.css]: resolved

File: WEATHERAPP/Components/Login/LoginPage.css (hunk 2 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/Login/LoginPage.css]: resolved

File: WEATHERAPP/Components/Login/LoginPage.css (hunk 3 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/Login/LoginPage.css:105]: resolved

File: WEATHERAPP/Components/Login/LoginPage.css (hunk 4 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/Login/LoginPage.css]: resolved

File: WEATHERAPP/Components/Login/LoginPage.html (hunk 5 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/Login/LoginPage.html:1]: resolved

File: WEATHERAPP/Components/Login/LoginPage.html (hunk 6 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/Login/LoginPage.html]: resolved

**Review by AkhilTrenser-123 — APPROVED** [review]

**Pull request merged**